tysnj/fitlit
# Feature/js refactor

## Is this a fix or a feature?
Fix

## What is the change?
Refactor the javascript, in all files but mainly scripts

## What does it fix?
Fixes formatting issues, combines redundant functions, makes sure all tests are still passing, removes unnecessary date params

## Where should the reviewer start?
git fetch `feature/JS-refactor`

## How should this be tested?
Make sure all functionality is still bug-free, run linter to check for formatting issues, run npm tests
